core/jazz_asset-event-handler/test/test.js

```diff
@@ -508,29 +515,68 @@ describe('jazz asset handler tests: ', function () {
       })
     });
 
-    it("should call processEvents",()=>{
-      processEventsStub = sinon.stub(index, "processEvents").resolves(result);
-      index.handler(event, context, (error, records)=>{
-        sinon.assert.calledOnce(processEventsStub);
-        processEventsStub.restore();
-      })
+    it('index should resolve for not interested events', function () {
+      var callback = (err, responseObj) => {
```

Comment:
i still don't see the change as per my previous comment

Reply:
Removed callback variable as it was not used.